tests/test_dpo_trainer.py

```diff
@@ -782,7 +782,8 @@ def test_dpo_lora_tags(self):
                 peft_config=lora_config,
             )
 
-            assert trainer.model.model_tags == trainer._tag_names
+            assert "dpo" in trainer.model.model_tags
```

Comment:
WDYT about gradually moving to using `unittest`'s methods for the tests (like we do in other HF libs)?

In this case, I think you could write:

```python
for tag in ["dpo", "trl"]:
    self.assertIn(tag, trainer.model.model_tags)
```

Reply:
#2137

Reply:
[65af404](https://github.com/huggingface/trl/pull/2123/commits/65af404b6ce79f2949596a755007ff8fbbcbfc8d)